Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/DirectILLSelfShielding.py

```diff
@@ -120,11 +120,6 @@ def PyInit(self):
         self.setPropertyGroup(common.PROP_SPARSE_INSTRUMENT_COLUMNS, PROPGROUP_SIMULATION_INSTRUMENT)
         self.setPropertySettings(common.PROP_SPARSE_INSTRUMENT_COLUMNS, EnabledWhenProperty(common.PROP_SIMULATION_INSTRUMENT,
                                  PropertyCriterion.IsEqualTo, common.SIMULATION_INSTRUMEN_SPARSE))
-        self.declareProperty(name=common.PROP_NUMBER_OF_SIMULATION_WAVELENGTHS,
```

Comment:
I think this property we still have to keep, but just ignore it. We should add in the doc of the property that it will not be used.

Reply:
Why do you want to keep this - is it to remove any problems from calls to the algorithm that use positional parameters?
It will only have any functional benefit if we also expose the associated "ResimulateTracksForDifferentWavelength" parameter. Even then it won't make any change to the results. That was why I decided to remove it.
I decided to keep it "NumberOfSimulationWavelengths" in the MonteCarloAbsorption algorithm but that was only because we may introduce some wavelength specific features in the generation of the tracks at some point in the future. Didn't seem worth propagating this decision out into all of the "wrapper" algorithms.

Reply:
Yes, I am pretty sure there are user scripts that specify this, so they will break down.